plugin/pkg/scheduler/factory/factory.go

```diff
@@ -257,6 +366,21 @@ func (c *ConfigFactory) updatePodInCache(oldObj, newObj interface{}) {
 	if err := c.schedulerCache.UpdatePod(oldPod, newPod); err != nil {
 		glog.Errorf("scheduler cache UpdatePod failed: %v", err)
 	}
+
+	// if the pod does not have binded node, updating equivalence cache is meaningless;
```

Comment:
Can you please split this into a separate function and just call it here?
e.g. invalidateCachedPredicatesOnUpdatePod or sth like that?